internal/profiles/service.go

```diff
@@ -276,6 +291,63 @@ func (p *profileService) UpdateProfile(
 	return profile, nil
 }
 
+func (p *profileService) PatchProfile(
+	ctx context.Context,
+	projectID uuid.UUID,
+	profileID uuid.UUID,
+	subscriptionID uuid.UUID,
```

Comment:
Do we want to allow `PATCH` with subscriptions?  If not, could we drop this argument?

Reply:
(It looks like we pass `uuid.Nil` as the argument to `UpdateProfile` from the other gRPC endpoint, so it looks like we the PATCH-on-Subscription codepath is effectively dead...)

Reply:
probably not, you're right. We'll likely to only update subscription profiles through a replace, not patch.